bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java

```diff
@@ -19,7 +19,7 @@
     /**
      * The version of modernizr
      */
-    String MODERNIZR_VERSION = "2.8.3-1";
+    String MODERNIZR_VERSION = "2.8.3";
```

Comment:
Same should be done in pom.xml: https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/80fa4d865f89eee8b606291d5757b442c109a904/pom.xml#L64